test/support/page_objects/visualize_page.js

```diff
@@ -492,7 +491,7 @@ export default class VisualizePage {
 
 
   // The current test shows dots, not a line.  This function gets the dots and normalizes their height.
-  getLineChartData(cssPart) {
```

Comment:
If this parameter isn't being used anymore, the functional tests appear to be still providing it, should we consider removing it there? 

https://github.com/elastic/kibana/blob/master/test/functional/apps/visualize/_line_chart.js#L65
https://github.com/elastic/kibana/blob/master/test/functional/apps/visualize/_line_chart.js#L95